package.json

```diff
@@ -279,13 +280,14 @@
     "bignumber.js": "^4.1.0",
     "bn.js": "^4.11.7",
     "bowser": "^2.11.0",
+    "btoa": "^1.2.1",
```

Comment:
Nit: If we set this during the unit test setup script, we can move this to `devDependencies` and avoid having this in our bundle

Reply:
removed from deps here: https://github.com/MetaMask/metamask-extension/pull/17652/commits/5bb475037056a114cec96db35fbc52e503a8fcc4